File: inspect_logo.js
const { Jimp } = require('jimp');

async function inspect() {
    const img = await Jimp.read('images/logo_raw.png');
    console.log('Image dimensions:', img.width, 'x', img.height);
    
    // Sample a few pixels from the corners and center
    const corners = [
        { x: 0, y: 0 },
        { x: img.width - 1, y: 0 },
        { x: 0, y: img.height - 1 },
        { x: img.width - 1, y: img.height - 1 },
        { x: Math.floor(img.width / 2), y: Math.floor(img.height / 2) }
    ];
    
    corners.forEach(p => {
        const hex = img.getPixelColor(p.x, p.y).toString(16).padStart(8, '0');
        console.log(`Pixel at (${p.x}, ${p.y}): #${hex}`);
    });
}

inspect().catch(console.error);

File: inspect_logo.py
import sys
from PIL import Image

def inspect():
    try:
        img = Image.open('images/logo_raw.png')
        print(f"Image dimensions: {img.width} x {img.height}")
        
        corners = [
            (0, 0),
            (img.width - 1, 0),
            (0, img.height - 1),
            (img.width - 1, img.height - 1),
            (img.width // 2, img.height // 2)
        ]
        
        # Convert image to RGBA to ensure we have pixel channels
        img_rgba = img.convert('RGBA')
        
        for x, y in corners:
            r, g, b, a = img_rgba.getpixel((x, y))
            # Jimp outputs hex in RRGGBBAA format.
            hex_color = f"{r:02x}{g:02x}{b:02x}{a:02x}"
            print(f"Pixel at ({x}, {y}): #{hex_color}")
            
    except Exception as e:
        print(f"Error: {e}", file=sys.stderr)

if __name__ == '__main__':
    inspect()
